src/relentless/simulate/hoomd.py

```diff
@@ -709,6 +719,38 @@ def _call_v2(self, sim):
             for analyzer in self.analyzers:
                 analyzer.pre_run(sim, self)
 
+            if isinstance(self.barostat, extent.Extent):
```

Comment:
In this case, the period of the updater should be set so that the action is taken immediately. I forget if that is `None` or something else.

Reply:
I checked the 2.9.7 documentation and to start immediately then `phase=-1`. Also, `scale_particles` defaults to `True` so I have removed that argument.

Reply:
I think you actually need `period=None` to have it act immediately (i.e., resize right away). Otherwise, the default period is 1, so it would try to update over time. Concurrently, you're also giving it a constant for the sizes, so these are probably essentially the same except that the updater will keep triggering over and over again.

So in summary, I think `period=None` when we just want to change the box size immediately (a single `Extent`), otherwise `period=1` to go between two boxes.

https://hoomd-blue.readthedocs.io/en/v2.9.7/module-hoomd-update.html#hoomd.update.box_resize